mozilla/foundation.mozilla.org
# updated/reordered CMS fields, removed price field, removed hint text

Closes #7614 

In this PR I did the following: 
- [x] Remove hint text (for easy scanability) 
- [x] Update titles so they match the [spreadsheet](https://docs.google.com/spreadsheets/d/1LBEYx6DrYClG0pFkfwLdCZN3KzHRuMgMTjZBqPGD2sg/edit#gid=1764292170) 
- [x] Remove Price field from CMS 
- [x] Use 'Yes' for Thumbs Up, 'No' for Thumbs Down, and 'Unknown' for Thumb Sideways
- [x] Reordered the CMS fields to align with the data entry sheet


### Test Pages:
**Link to test General Product Page in the CMS:**https://foundation-s-7614-updat-nn3k3e.herokuapp.com/cms/pages/73/edit/
**Link to test Software Product Page in the CMS:**https://foundation-s-7614-updat-nn3k3e.herokuapp.com/cms/pages/74/edit/


**Steps to test:**

1. Using the credentials in the #mofo-ra-foundation channel, please visit both pages linked above to view the updated names and order in the CMS.
2. The order of the fields should now match the order in https://docs.google.com/spreadsheets/d/1LBEYx6DrYClG0pFkfwLdCZN3KzHRuMgMTjZBqPGD2sg/edit#gid=1764292170.
3. If everything looks as expected, testing is complete!
